chai_research/qlora/requirements.txt

```diff
@@ -0,0 +1,45 @@
+accelerate @ git+https://github.com/huggingface/accelerate.git@21d127334ebbcdd4f2f0695a5b36753dae6d747b
```

Comment:
When I followed the readme to install requirements.txt. Later it complains don't have  (from scipy) (1.25.2). Maybe we need to add it here?